data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+	npc evade
+		government "Cerberus"
+		personality disables plunders vindictive
+		system "Almaaz"
+		fleet "Small Northern Pirates" 2
+	npc evade
+		government "Cerberus"
+		personality disables plunders vindictive waiting
+		system "Almaaz"
+		fleet "Large Northern Pirates"
+
+# Additional ships will spawn if you brought a JD with you.
+mission "<uhai> 3: Jump Drive Vault: Attracted Attention"
+	landing
+	invisible
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "<uhai> 3: Recover Jump Drives: active"
+		has "outfit (installed): Jump Drive"
+	on enter "Almaaz"
+		"unfettered rep: Si-Ra" -= 2
+		dialog `The Unfettered "pirate" spits a string of old-timey nautical curses, but one word, which you can barely hear him mutter, stands out: "Cerberus." He then comes closer to address you directly, "There be even more of these scurvy bilge rats than I expected. I warned ye about coming here with a jump drive!"`
+	npc evade
+		government "Cerberus"
+		system "Almaaz"
+		personality disables plunders vindictive
+		fleet "Large Northern Pirates"
+
+
+
+# This variant has no reverse thrusters to make it easier to scan it and to attract more attention,
+# especially with the keystones.
+ship "Geocoris" "Geocoris (Keystones)"
+	crew 56
+	outfits
+		"Hai Tracker Pod" 2
+		"Hai Tracker" 112
+		"Chameleon Anti-Missile" 3
+		"Hai Cuttlefish Jammer" 4
+		"Boulder Reactor"
+		"Hai Valley Batteries"
+		"Hai Diamond Regenerator"
+		"Hai Corundum Regenerator"
+		"Hai Williwaw Cooling" 4
+		`"Biroo" Atomic Thruster`
+		`"Biroo" Atomic Steering`
+		Hyperdrive
+		"Pulse Rifle" 56
+	cargo
+		outfits
+			"Quantum Keystone" 123
+
+mission "<uhai> 4: Scan Geocoris"
+	name `Scan the Geocoris`
+	description `Scan the cargo of the Hai Geocoris in Ya Hai with the Value Detector, then return to <destination> with the scanner logs. You may abort this, at a cost.`
+	source "Firelode"
+	waypoint "Ya Hai"
+	to offer
+		random < 60
+		has "event: unfettered: access to Mera's lab"
+	on offer
+		conversation
+			`As you wander through the spaceport, you find a market with a limited diversity of goods, although some of it seems to be completely alien, even by Hai standards. You can even spot human food and technology, although those are the only recent items here. Most of it is very ancient, and you doubt everything is functional...`
+			`	Most Unfettered here are just busy buying what little food there is, that is if they can even afford it. You see two of them disputing, probably over the cost of some item, but decide not to intervene given how they generally respond to your mere presence.`
+			`	An Unfettered approaches you. At first, you wonder why the Hai happens to be coming towards you in particular, but then you realize that you naturally stand out in the crowd. There are not many non-Hai on the Unfettered worlds.`
+			`	"Captain <last>! I am Mera, an engineer. We have an easy job for you, if you are interested," she says. "It would pay well," she adds.`
```

Comment:
```suggestion
			`	"Captain <last>! I am Mera, an engineer. We have an easy job for you, if you are interested," she says, and adds, "It would pay well."`
```
Duplicate she.